app/services/backup.tsx

```diff
@@ -151,38 +166,24 @@ class Backup {
                                 console.log("error");
                             }
                             return "Imported";
+
                         }
                     }
                     else {
-                        await setActiveChild(langCode, '', dispatch, childAge, false);
-                        navigation.navigate('LoadingScreen', {
-                            apiJsonData: [],
-                            prevPage: 'ImportScreen'
-                        });
-                        try {
-                            Realm.deleteFile({ path: RNFS.TemporaryDirectoryPath + '/' + 'user1.realm' });
-                        } catch (error) {
-                            console.log("error");
-                        }
-                        return "Imported";
-
+                        return new Error('No Data');
                     }
-                }
-                else {
-                    return new Error('No Data');
-                }
-            }).catch(error => {
-                console.log("error-", error);
-                return new Error('No Import Succeded');
-            })
+                }).catch(error => {
+                    console.log("error-", error);
+                    return new Error('No Import Succeded');

```

Comment:
1. It should be "Succeeded" not "Succeded".
2. Instead of returning error you should throw.

```
.catch(error => {
    console.error("Error occurred:", error);
    throw new Error('No Import Succeeded');
});
```